converter/internal/otelcolconvert/converter_k8sattributesprocessor.go

```diff
@@ -80,6 +80,11 @@ func toExclude(cfg k8sattributesprocessor.ExcludeConfig) k8sattributes.ExcludeCo
 	}
 
 	for _, c := range cfg.Pods {
+		// These are default excludes from upstream opentelemetry-collector-contrib, we don't have to convert them.
+		// Source: https://github.com/open-telemetry/opentelemetry-collector-contrib/blame/main/processor/k8sattributesprocessor/factory.go#L21
+		if c.Name == "jaeger-agent" || c.Name == "jaeger-collector" {
+			continue
+		}
```

Comment:
I don't think this is quite right; my read on what upstream is doing is that if excludes _aren't_ set, then it defaults to `jaeger-agent` and `jaeger-collector`, not that they are always excluded.

Reply:
Yes, I misinterpreted the intent and made changes in wrong place as well. I've made changes in the component in HEAD instead. PTAL if you have time to do so :D